src/optable.ts

```diff
@@ -465,6 +465,19 @@ optable[opcodes.STORE_NAME] = function(f: Py_FrameObject) {
     f.locals[name.toString()] = val;
 }
 
+optable[opcodes.UNPACK_SEQUENCE] = function(f: Py_FrameObject) {
+    var val = f.pop();
+    if(val === undefined) {
```

Comment:
The assertion should check if the method `val.__getitem__` is undefined, not `val` itself. If that item is missing, then we are not dealing with a list type. We should never have an undefined value on the stack. :)